crates/db_schema/src/impls/actor_language.rs

```diff
@@ -290,6 +292,9 @@ pub async fn default_post_language(
 
   if intersection.len() == 1 {
     Ok(Some(intersection[0]))
+  } else if intersection.len() == 2 && intersection.contains(&UNDETERMINED_ID) {
+    intersection.retain(|i| i != &UNDETERMINED_ID);
```

Comment:
Seems like you already know its going to be at `intersection.get(1)`, if these are sorted, but this works.

Reply:
Right that should also work. Not sure which way is better.